pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AvroSchema.java

```diff
@@ -88,14 +97,19 @@ public boolean supportSchemaVersioning() {
         return new AvroSchema<>(parseSchemaInfo(schemaDefinition, SchemaType.AVRO));
     }
 
+    public static <T> AvroSchema<T> ofWithClass(SchemaDefinition<T> schemaDefinition) {
```

Comment:
Do we need a separate static method? Why can we just check if schemaDefinition has POJO? If it has POJO, when we create a schema instance, we should include the POJO.

Reply:
Yes, it's more reasonable.